src/systems/filecoin_markets/deal/_index.md

```diff
@@ -23,8 +23,7 @@ Filecoin Storage Market Deal Flow
 
 # Receive Challenge
 
-- 5. Miners enter the `Challenged` status whenever `NotifyOfPoStChallenge` is called by the chain. Miners will then have X Epoch as the ProvingPeriod to submit a successful PoSt before `
-CheckPoStSubmissionHappened` is called by the chain. Miners can only get out the challenge with `SubmitPoSt` or `onMissedPoSt`.
+- 5. Miners enter the `Challenged` status whenever `NotifyOfPoStChallenge` is called by the chain. Miners will then have X Epoch as the ProvingPeriod to submit a successful PoSt before `CheckSurprisePoStSubmissionHappened` is called by the chain. Miners can only get out the challenge with `SubmitSurprisePoSt` or `onMissedSurprisePoSt`.
```

Comment:
Miner also enters and exits the Challenged status in the same epoch when generating ElectionPoSt. Is this correct?

Reply:
yes exactly